merlin:
  hardware_interface:
    loop_hz: 20 # hz
    joints:
      - waist
      - shoulder
      - elbow
      - wrist_rotation
      - wrist_flex
      - hand_roll